<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1123</width>
    <height>717</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DCM Piezo Rocking Curve</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>30</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>DCM Piezo Rocking Curve </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>start vol</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>step vol</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>100</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>end vol</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>100</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>integral time(s)</string>
   </property>
  </widget>
  <widget class="PyDMLineEdit" name="PyDMLineEdit">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>130</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="alarmSensitiveBorder" stdset="0">
    <bool>false</bool>
   </property>
   <property name="channel" stdset="0">
    <string>X09B1:EH:NCT:SetTimer.A</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>I0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>260</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>I1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStart">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>190</y>
     <width>92</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>100</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>settling time(s)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStop">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>190</y>
     <width>92</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>next vol</string>
   </property>
  </widget>
  <widget class="PyDMLineEdit" name="PyDMLineEdit_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="alarmSensitiveBorder" stdset="0">
    <bool>false</bool>
   </property>
   <property name="channel" stdset="0">
    <string>X09B1:RC:Start</string>
   </property>
  </widget>
  <widget class="PyDMLineEdit" name="PyDMLineEdit_5">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="alarmSensitiveBorder" stdset="0">
    <bool>false</bool>
   </property>
   <property name="channel" stdset="0">
    <string>X09B1:RC:Step</string>
   </property>
  </widget>
  <widget class="PyDMLineEdit" name="PyDMLineEdit_6">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>130</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="alarmSensitiveBorder" stdset="0">
    <bool>false</bool>
   </property>
   <property name="channel" stdset="0">
    <string>X09B1:RC:End</string>
   </property>
  </widget>
  <widget class="PyDMLabel" name="PyDMLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>200</y>
     <width>73</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="alarmSensitiveBorder" stdset="0">
    <bool>false</bool>
   </property>
   <property name="channel" stdset="0">
    <string>X09B1:RC:Next</string>
   </property>
  </widget>
  <widget class="PyDMLabel" name="PyDMLabel_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>260</y>
     <width>73</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="alarmSensitiveBorder" stdset="0">
    <bool>false</bool>
   </property>
   <property name="channel" stdset="0">
    <string>X09B1:EH:NCT:V0</string>
   </property>
  </widget>
  <widget class="PyDMLabel" name="PyDMLabel_3">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>260</y>
     <width>73</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="alarmSensitiveBorder" stdset="0">
    <bool>false</bool>
   </property>
   <property name="channel" stdset="0">
    <string>X09B1:EH:NCT:V1</string>
   </property>
   <property name="displayFormat" stdset="0">
    <enum>PyDMLabel::Default</enum>
   </property>
  </widget>
  <widget class="PyDMLineEdit" name="PyDMLineEdit_2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>130</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="alarmSensitiveBorder" stdset="0">
    <bool>false</bool>
   </property>
   <property name="channel" stdset="0">
    <string>X09B1:RC:SettlingTime</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>100</y>
     <width>351</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>file save path: /home/bl09b1/daq-data</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>130</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>file name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fileName">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>130</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="PyDMTimePlot" name="PyDMTimePlot">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>501</width>
     <height>351</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="showXGrid">
    <bool>true</bool>
   </property>
   <property name="showYGrid">
    <bool>true</bool>
   </property>
   <property name="backgroundColor">
    <color>
     <red>255</red>
     <green>255</green>
     <blue>255</blue>
    </color>
   </property>
   <property name="axisColor">
    <color>
     <red>0</red>
     <green>0</green>
     <blue>0</blue>
    </color>
   </property>
   <property name="curves">
    <stringlist>
     <string>{&quot;channel&quot;: &quot;X09B1:EH:NCT:V0&quot;, &quot;name&quot;: &quot;&quot;, &quot;color&quot;: &quot;red&quot;, &quot;lineStyle&quot;: 1, &quot;lineWidth&quot;: 2, &quot;symbol&quot;: null, &quot;symbolSize&quot;: 10}</string>
    </stringlist>
   </property>
   <property name="timeSpan">
    <double>60.000000000000000</double>
   </property>
  </widget>
  <widget class="PyDMTimePlot" name="PyDMTimePlot_2">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>290</y>
     <width>501</width>
     <height>351</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="showXGrid">
    <bool>true</bool>
   </property>
   <property name="showYGrid">
    <bool>true</bool>
   </property>
   <property name="backgroundColor">
    <color>
     <red>255</red>
     <green>255</green>
     <blue>255</blue>
    </color>
   </property>
   <property name="axisColor">
    <color>
     <red>0</red>
     <green>0</green>
     <blue>0</blue>
    </color>
   </property>
   <property name="curves">
    <stringlist>
     <string>{&quot;channel&quot;: &quot;X09B1:EH:NCT:V1&quot;, &quot;name&quot;: &quot;&quot;, &quot;color&quot;: &quot;red&quot;, &quot;lineStyle&quot;: 1, &quot;lineWidth&quot;: 2, &quot;symbol&quot;: null, &quot;symbolSize&quot;: 10}</string>
    </stringlist>
   </property>
   <property name="timeSpan">
    <double>60.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>motor readback</string>
   </property>
  </widget>
  <widget class="PyDMLabel" name="PyDMLabel_4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>170</y>
     <width>73</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="alarmSensitiveBorder" stdset="0">
    <bool>false</bool>
   </property>
   <property name="channel" stdset="0">
    <string>IOC:m1.RBV</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PyDMTimePlot</class>
   <extends>QGraphicsView</extends>
   <header>pydm.widgets.timeplot</header>
  </customwidget>
  <customwidget>
   <class>PyDMLabel</class>
   <extends>QLabel</extends>
   <header>pydm.widgets.label</header>
  </customwidget>
  <customwidget>
   <class>PyDMLineEdit</class>
   <extends>QLineEdit</extends>
   <header>pydm.widgets.line_edit</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
